tests/baselines/reference/jsxNamespaceImplicitImportJSXNamespaceFromPragmaPickedOverGlobalOne.types

```diff
@@ -3,7 +3,7 @@ export = React;
 >React : any
 
 export as namespace React;
->React : any
+>React : error

```

Comment:
Hm, is this one expected? It also seems to be like this in the [`reactTransitiveImportHasValidDeclaration test`](https://github.com/microsoft/TypeScript/blob/566faa7a01f3d483f032fda4777dae186eccbb19/tests/baselines/reference/reactTransitiveImportHasValidDeclaration.types#L10) which I haven't touched as part of this PR.

Reply:
I wanna say "yes", since this location doesn't really have a type. It displays as `any` unless the test has no error diagnostics, in which case it's printed to the baselines as an `error` type to indicate a location we're incorrectly using the `errorType` instead of the `anyType` in the `getTypeAtLocation` API. Its not much of an issue in practice.